oauthenticator/tests/test_mediawiki.py

```diff
@@ -27,7 +27,7 @@ def post_token(request, context):
                         'iss': 'https://meta.wikimedia.org',
                         'iat': time.time(),
                         'nonce': request_nonce,
-                    }, 'client_secret')
+                    }, 'client_secret').encode("utf8")
```

Comment:
This should be unrelated to the LANG env, which Python does not generally take into account for encoding except when producing terminal output. The error here was a TypeError because str is not allowed at all, only None or Bytes, so which encoding is used is not relevant. I think this was just a change in the requests_mock or requests version being strict about types.